activemodel/lib/active_model/attribute_methods.rb

```diff
@@ -361,6 +362,11 @@ def undefine_attribute_methods
         attribute_method_patterns_cache.clear
       end
 
+      def local_attribute_aliases
```

Comment:
There is probably a better name for this, but this was the best I could come up for now.